src/applications/facility-locator/components/SearchResult.jsx

```diff
@@ -3,26 +3,38 @@ import PropTypes from 'prop-types';
 import LocationInfoBlock from './search-results/LocationInfoBlock';
 import LocationPhoneLink from './search-results/LocationPhoneLink';
 import LocationDirectionsLink from './search-results/LocationDirectionsLink';
+import LocationAddress from './search-results/LocationAddress';
 
 // revert to stateless component given: 19fd5178f
 const SearchResult = ({ result, query }) => (
-  <div className="facility-result" id={result.id}>
-    <LocationInfoBlock location={result} from={'SearchResult'} query={query} />
-    <LocationDirectionsLink
-      location={result}
-      from={'SearchResult'}
-      query={query}
-    />
-    <LocationPhoneLink location={result} from={'SearchResult'} query={query} />
-    {query.facilityType === 'urgent_care' &&
-      query.serviceType === 'NonVAUrgentCare' && (
-        <p>
-          {' '}
-          Before going to a clinic for urgent care, please call the facility to
-          confirm that it's open and able to provide the care you need.{' '}
-        </p>
-      )}
-  </div>
+  <li className="facility-result" id={result.id}>
+    <dl>
+      <LocationInfoBlock
+        location={result}
+        from={'SearchResult'}
+        query={query}
+      />
+      <LocationAddress location={result} />
+      <LocationDirectionsLink
+        location={result}
+        from={'SearchResult'}
+        query={query}
+      />
+      <LocationPhoneLink
+        location={result}
+        from={'SearchResult'}
+        query={query}
+      />
+      {query.facilityType === 'urgent_care' &&
+        query.serviceType === 'NonVAUrgentCare' && (
```

Comment:
Are there constants that could be used for these comparisons instead of floating strings?